### PR TITLE
Adds the menu

### PR DESCRIPTION
**Pull request by moby-dicc**
Adds a button to show the various options and credits

### CODE REVIEW

**Comment by aquibbaig** [comment]
@moby-dicc Some initial comments on your work:
1. How does a user go back from a certain dialog box. (For example: I am in the About and I would like to either go back or close the dialog)
2. The order of the settings menu should be as follows:
 - Profile
 - Settings
 - How to play
 - About

File: lib/components/appbar.dart [hunk 1]
```diff
@@ -8,7 +9,400 @@ class GlobalAppBar extends StatefulWidget implements PreferredSizeWidget  {
...
+  Widget title =Row( 
+    children: <Widget>[
+       Align(
+    alignment: Alignment.centerLeft,
+    child: BackButton()
+      ),
+       Align(
+         alignment: AlignmentDirectional.center,
+      child:Container(
+    margin: const EdgeInsets.only(left:40.0),
```
Comment (aquibbaig) [inline lib/components/appbar.dart:39]: Add Media query for every constant used.

File: lib/components/appbar.dart [hunk 2]
```diff
@@ -8,7 +9,400 @@ class GlobalAppBar extends StatefulWidget implements PreferredSizeWidget  {
...
+      onTap: () {  
+      Navigator.of(context).pop();  
+     }, 
+    ), 
+  );
+  AlertDialog alert = AlertDialog(  
+    actions: [  
+      title, profile, settings, howtoplay, about,
+    ],  
+  );
```
Comment (aquibbaig) [inline lib/components/appbar.dart:397]: Dialog Box

Some design improvements:
- Give rounded border BoxDecoration
- Add little more space between the items, add space from left and right.
- "Options" is not centrally aligned
- Add more space for the content (make dialog bigger)
-

File: lib/components/credits.dart [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+                      launch('https://zairza.in/');
+                          },
+                    child: Container(                 
+                    //margin:EdgeInsets.fromLTRB(1.0,10.0,150.0,10.0) ,
+                    decoration: BoxDecoration(
+                      image: DecorationImage(
+                        image: AssetImage(
+                        'assets/zairza_logo.jpg'),
+                        fit: BoxFit.fill
+                            ),
```
Comment (aquibbaig) [inline lib/components/credits.dart:68]: Screenshot 2020-07-25 at 3 55 03 PM

The design does not look good IMO, to make a rather consistent design make this the exact size of the "options" dialog and scroll for more details. Add proper spaces and padding for the content/make dialog bigger.

File: lib/components/appbar.dart [hunk 4]
```diff
@@ -8,7 +9,400 @@ class GlobalAppBar extends StatefulWidget implements PreferredSizeWidget  {
...
+      child:Container(
+        height: 50.0,
+        child: Material(
+        borderRadius: BorderRadius.circular(10.0),
+        shadowColor: Color(0xFF6ad1ff),
+        //color: Color(0xFF039be5),
+        elevation: 7.0,
+        child: Center(
+          child: Text(
+            'How To Play',
```
Comment (aquibbaig) [inline lib/components/appbar.dart:379]: "How to play" is not created yet I guess

**Comment by moby-dicc** [comment]
@aquibbaig Please review the changes and make necessary comments.

**Comment by aquibbaig** [comment]
@moby-dicc Initial comments:

- How does the user exit the "Instructions" dialog in between
- Why do two dialog still show up when I click "About". Just display the developers page, not the dialogs.
- Reduce the size of icons and texts in "Instructions".

**Comment by aquibbaig** [comment]
@moby-dicc is this ready for review?

**Comment by moby-dicc** [comment]
> @moby-dicc is this ready for review?

Yes it is.

File: pubspec.yaml [hunk 5]
```diff
@@ -77,6 +81,7 @@ flutter:
     - assets/frame_blue.png
     - assets/frame_purple.png
     - assets/game_mat_basic.png
+    - assets/zairza_logo.jpg
   #  - images/a_dot_ham.jpeg
```
[the remark this answers is not in the document]
Comment (moby-dicc) [reply pubspec.yaml:85]: I've used the logo as a link to www.zairza.in

**Pull request merged**